packages/timeline-state-resolver/src/integrations/vmix/index.ts

```diff
@@ -137,31 +131,25 @@ export class VMixDevice extends DeviceWithState<VMixStateExtended, DeviceOptions
 			if (d.message !== 'Completed') this.emit('debug', d)
 			if (d.command === 'XML' && d.body) {
 				this._vmix.parseVMixState(d.body)
-
 				if (!this._initialized) {
-					const time = this.getCurrentTime()
-					let state = this._getDefaultState()
-					state = deepMerge<VMixStateExtended>(state, { reportedState: this._vmix.state })
-					this.setState(state, time)
 					this._initialized = true
 					this.emit('connectionChanged', this.getStatus())
 					this.emit('resetResolver')
 				}
-			} else if (d.response === 'ER' && FILE_NOT_FOUND_REGEX.test(d.message)) {
-				this._changeTrackedStateFromIncomingData(d)
 			}
 		})
 		// this._vmix.on('debug', (...args) => this.emitDebug(...args))
 
 		this._vmix.connect()
 
-		if (typeof options.retryInterval !== 'number' || options.retryInterval > 0) {
-			this._retryTime = options.retryInterval ?? DEFAULT_MEDIA_RETRY_INTERVAL
-			this._retryTimeout = setTimeout(() => this._assertIntendedState(), this._retryTime)
+		if (typeof options.pollInterval !== 'number' || options.pollInterval > 0) {
+			this._pollTime = options.pollInterval ?? DEFAULT_VMIX_POLL_INTERVAL
```

Comment:
This logic is pretty hard to follow.

```suggestion
		this._pollTime = options.pollInterval ?? DEFAULT_VMIX_POLL_INTERVAL // options.pollInterval  === 0 disables the polling
		if (this._pollTime  > 0 ) {
			
```

Reply:
Didn't end up with the exact same implementation.

One side effect of the proposed implementation is that non-numeric numbers will disable all polling, same as <= 0 nums.

If we want this case to fall to the default we need to unwrap this into two IFs.

Reply:
A side-effect of (both the renaming and this change) is that we now have two blocks/implementations of similar-but-different code for the same prop/mechanism between CasparCG and vMix.